codyze-backends/cpg/build.gradle.kts

```diff
@@ -13,3 +14,28 @@ dependencies {
 
     testImplementation(libs.mockk)
 }
+
+publishing {
+    publications {
+        create<MavenPublication>("CodyzeBackendsCpg") {
+            from(components["java"])
+            artifact(tasks.named("dokkaHtmlJar")) // docs generated with Dokka
+            pom {
+                name.set("Codyze Backends CPG")
+                description.set("CPG backend for Codyze")
+                url.set("https://www.codyze.io/")
+                licenses {
+                    license {
+                        name.set("Apache License, Version 2.0")
+                        url.set("https://www.apache.org/licenses/LICENSE-2.0.txt")
+                    }
+                }
+                scm {
+                    connection.set("scm:git:git://github.com/Fraunhofer-AISEC/codyze.git")
+                    developerConnection.set("scm:git:ssh://git@github.com/Fraunhofer-AISEC/codyze.git")
+                    url.set("https://github.com/Fraunhofer-AISEC/codyze")
+                }
+            }
+        }
+    }
+}
```

Comment:
Similarly to the [common-conventions plugin](https://github.com/Fraunhofer-AISEC/cpg/blob/main/buildSrc/src/main/kotlin/cpg.common-conventions.gradle.kts#L53) in the CPG project, we could get rid of most of the code duplication here.
In the modules that should be published, this prepared publication can be used like [so](https://github.com/Fraunhofer-AISEC/cpg/blob/main/cpg-analysis/build.gradle.kts#L31).

Reply:
I've created a default publication configuration that is adjusted in each module. However, the current names including the artifacts may collide with other packages, e.g., our Codyze backend for the CPG creates the artifact `cpg-.*` under `de.fraunhofer.aisec`.

Do you know a simple way to use, for example, the full module path as publication name (e.g., `codyze-backends:cpg`)?